conferences.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: Where We'll Be
+permalink: /wherewellbe/
+---
+
+Here is a list of upcoming events where you can see some of us speak:
+
+<ul class="events">
+{% for event in site.data.conferences %}
```

Comment:
Need to filter this to not show past events so we don't need to keep coming back and deleting them. Also, if possible, sort based on date after filtering. This probably means we need a start_date parameter for the filter and sort since the display date might not be a well-formatted date.